src/rgw/rgw_op.cc

```diff
@@ -1027,13 +1027,15 @@ int RGWGetObj::handle_slo_manifest(bufferlist& bl)
 
   for (const auto& entry : slo_info.entries) {
     const string& path = entry.path;
-    const size_t pos = path.find('/', 1); /* skip first / */
-    if (pos == string::npos) {
+    /* If the path starts with slashes, strip them all. */
+    const size_t pos_init = path.find_first_not_of('/');
+    const size_t pos_sep = path.find('/', pos_init);
```

Comment:
This might be true with the accuracy to implementation's bugs (see [this question on StackOverflow](http://stackoverflow.com/questions/1011790/why-does-stdstring-findtext-stdstringnpos-not-return-npos)).

Pushed a reworked version with the extra check on board.